app/Filament/Server/Widgets/ServerConsole.php

```diff
@@ -103,6 +104,50 @@ public function enter(): void
         }
     }
 
+    public function cpuUsage(): string
+    {
+        if ($this->server->condition === 'offline') {
```

Comment:
At the top make sure to import the class:

```php
use App\Enums\ContainerStatus;
```

```suggestion
        if (ContainerStatus::tryFrom($this->server->retrieveStatus()) === ContainerStatus::Offline) {
```